ansible/roles/ocp4-workload-security-compliance-lab/tasks/workload.yml

```diff
@@ -44,7 +21,7 @@
 - name: create temporary git directory
   tempfile:
     state: directory
-    path: "{{tmp_dir}}"
+    path: "{{ tmp_dir }}"
```

Comment:
see if this is already used for infra tasks that you are copying files to and whether it is needed again here